convince-my-boss.html

```diff
@@ -0,0 +1,171 @@
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="PyCon India, the premier conference in India on using and developing the Python programming language is conducted annually by the Python developer community and it attracts the best Python programmers from across the country and abroad.">
+    <meta name="author" content="">
+    <title>PyCon India 2018 in Hyderabad | October 5th to 9th </title>
+
+    <!-- canonical URL -->
+    <link rel="canonical" href="https://in.pycon.org/2018/faq.html">
+
+    <!-- Bootstrap Core CSS -->
+    <link href="custom/css/bootstrap.min.css" rel="stylesheet" type="text/css">
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-111617773-1"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', 'UA-111617773-1');
+    </script>
+
+
+    <!-- Fonts -->
+    <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
+    <link href="custom/css/animate.css" rel="stylesheet" />
+    <link href="custom/css/style.css" rel="stylesheet">
+    <link href="custom/css/scroll.css" rel="stylesheet">
+    <link href="custom/css/reset.css" rel="stylesheet">
+    <link href="custom/css/default.css" rel="stylesheet">
+    <link href="custom/css/buttons.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat+Alternates:300,400,500,600" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet">
+</head>
+
+<body id="page-top" data-spy="scroll" data-target=".navbar-custom">
+    <!-- Preloader -->
+    <!-- <div id="preloader">
+    <div id="load"></div>
+  </div> -->
+    <nav class="navbar navbar-custom navbar-fixed-top" style="-webkit-box-shadow: 0 1px 30px rgba(0, 0, 0, .1);">
+        <div class="container">
+            <img src="img/banner-logo.png" class="logo" alt="banner-logo" id="logo" style="width: 150px;">
+            <div class="navbar-header">
+                <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+                    <span class="sr-only">Toggle navigation</span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                </button>
+            </div>
+
+            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+                <ul class="nav nav-justified">
+                    <li><a href="index.html#home">Home</a></li>
+                    <li><a href="index.html#journey">Journey</a></li>
+                    <!-- <li><a href="index.html">Speakers</a></li> -->
```

Comment:
Uncomment this now as the Keynote speaker PR is merged.

Reply:
I mentioned the answer to this in the previous review that was done by you.
I will be fixing all of them at once in #110, that would be easy to track in a single PR.

Reply:
Ok that will be fine.